src/libstore/globals.hh

```diff
@@ -295,6 +295,9 @@ public:
         "Nix store has a valid signature (that is, one signed using a key "
         "listed in 'trusted-public-keys'."};
 
+    Setting<StringSet> extraPlatforms{this, StringSet{}, "build-extra-platforms",
```

Comment:
IMO this should default ti `i686-linux` on `x86_64-linux`

Reply:
Makes sense.

Reply:
Shouldn't we have all the platforms from https://github.com/NixOS/nix/pull/1916/files#diff-81a5a969aed0f3f9d6dc4e99b92925cdL60 ?

Reply:
I'm not sure having all of them is a good idea, given how unreliable actually building them is: building armv6 stuff on an armv7 machine will often result in code that doesn't actually run on armv6 because of annoying autodetection in various build systems. Users who want to risk that can easily set this setting as appropriate.